src/test/java/uk/gov/pay/webhooks/message/dao/WebhookMessageDaoTest.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.pay.webhooks.message.dao;
+
+import io.dropwizard.testing.junit5.DAOTestExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import uk.gov.pay.webhooks.deliveryqueue.dao.WebhookDeliveryQueueDao;
+import uk.gov.pay.webhooks.deliveryqueue.dao.WebhookDeliveryQueueEntity;
+import uk.gov.pay.webhooks.eventtype.dao.EventTypeEntity;
+import uk.gov.pay.webhooks.message.dao.entity.WebhookMessageEntity;
+import uk.gov.pay.webhooks.webhook.dao.WebhookDao;
+import uk.gov.pay.webhooks.webhook.dao.entity.WebhookEntity;
+
+import java.time.Instant;
+import java.time.InstantSource;
+import java.util.Date;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+class WebhookMessageDaoTest {
+    public DAOTestExtension database = DAOTestExtension.newBuilder()
+            .addEntityClass(WebhookEntity.class)
+            .addEntityClass(WebhookMessageEntity.class)
+            .addEntityClass(WebhookDeliveryQueueEntity.class)
+            .addEntityClass(EventTypeEntity.class)
+            .build();
+
+    private WebhookDao webhookDao;
+    private WebhookMessageDao webhookMessageDao;
+    private WebhookDeliveryQueueDao webhookDeliveryQueueDao;
+    String webhookExternalId = "webhook-external-id";
+
+    @BeforeEach
+    public void setUp() {
+        webhookDao = new WebhookDao(database.getSessionFactory());
+        webhookMessageDao = new WebhookMessageDao(database.getSessionFactory());
+        webhookDeliveryQueueDao = new WebhookDeliveryQueueDao(database.getSessionFactory(), new InstantSource() {
+            @Override
+            public Instant instant() {
+                return Instant.now();
+            }
+        });
+    }
```

Comment:
we can simplify the creation of the InstantSource:
```suggestion
        webhookDeliveryQueueDao = new WebhookDeliveryQueueDao(database.getSessionFactory(), InstantSource.fixed(Instant.now()));
```

Reply:
Good catch, this is much nicer 👍